invoice_split/wizards/invoice_split.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2018 Vauxoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class AccountInvoiceSplit(models.TransientModel):
+    _name = "account.invoice.split"
+    _description = "Invoice Splitting"
+
+    invoice_id = fields.Many2one('account.invoice', String="Invoice to Split")
+    new_invoice_id = fields.Many2one('account.invoice')
+    percent = fields.Float(
+        string="Percent to Pay", default=50.0,
+        help="Percent of the amount that will be paid on the current invoice")
+    create_invoice = fields.Boolean(
+        string="Create New Invoice", default=True,
+        help="Specifies whether a new invoice will be created in draft mode "
+        "with the remaining uncovered amount")
+
```

Comment:
@luisg123v Check if the invoice is in draft state, something like this:
```suggestion
    @api.model
    def default_get(self, fields):
        rec = super(AccountInvoiceSplit, self).default_get(fields)
        active_ids = self.env.context.get('active_ids')
        active_model = self.env.context.get('active_model')

        # Check for selected invoices ids
        if not active_ids or active_model != 'account.invoice':
            return rec

        invoices = self.env['account.invoice'].browse(active_ids)

        # Check invoices are draft
        if any(invoice.state != 'draft' for invoice in invoices):
            raise UserError(_("You can only split draft invoices"))
```